appveyor.yml

```diff
@@ -0,0 +1,30 @@
+version: 1.0.{build}
```

Comment:
I think we don't need that. AppVeyor is already integrated and may soon be dropped in favor of Github Actions

Reply:
done